apps/gp2-frontend/src/users/country-codes-suggestions.ts

```diff
@@ -0,0 +1,1213 @@
+export const countryCodesSuggestions = [
+  {
+    name: 'Afghanistan',
+    dialCode: '+93',
+    code: 'AF',
```

Comment:
Do we need the code?

Reply:
we don't use the codes
I can clean them up